tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
It seems that you are using same template in every module, so I think people just need to read code once to be able to handle it, so you are right